sdks/python/setup.py

```diff
@@ -90,6 +90,7 @@ def get_version():
     'httplib2>=0.8,<0.10',
     'mock>=1.0.1,<3.0.0',
     'oauth2client>=2.0.1,<4.0.0',
+    'protobuf==3.0.0',
```

Comment:
I worry about what will happen when, say, tensorflow (or any other library) moves to upgrade the protobuf they use.  The upgrade / compatibility story may be painful, since we would need to coordinate an upgrade with their release.  But I don't see a good way around this here.